blackbox/docs/sql/reference/create_function.txt

```diff
@@ -0,0 +1,65 @@
+.. highlight:: psql
+.. _ref-create-function:
+
+===============
+CREATE FUNCTION
+===============
+
+Create a new function.
+
+Synopsis
+========
+
+::
+
+    CREATE [OR REPLACE] FUNCTION [ schema_name. ] function_name
+        ( [ [arg_name] arg_type ] [, ...] ] )
+    RETURNS return_type
+    { LANGUAGE language_name
+      | CALLED ON NULL INPUT | RETURNS NULL ON NULL INPUT | STRICT
+      | AS 'definition'
+    }
+
+Description
+===========
+
+``CREATE FUNCTION`` creates a new user defined function.
+
+``CREATE OR REPLACE FUNCTION`` will either create a new function, or replace an
+existing function.
+
+Optionally, a function name can contain a schema name. If the schema is not
+defined, the function is created in the default ``doc`` schema.
```

Comment:
afaik the default is not always ``doc``. The default depends on the user session